### DISCUSSION TITLE
Can't update oh-my-posh to 24.13

### DISCUSSION THREAD

**Discussion by PilotBob** [JanDeDobbeleer/oh-my-posh#6015] [General]
I am trying to update oh-my-posh. It looks like the most recent version doesn't have the same file names or something. 

I get a failure trying to use scoop:

scoop status
Scoop is up to date.
WARN  error: The remote server returned an error: (404) Not Found.

Name       Installed Version Latest Version Missing Dependencies Info
----       ----------------- -------------- -------------------- ----
oh-my-posh 21.27.0                                               Manifest removed

 scoop update oh-my-posh
WARN  error: The remote server returned an error: (404) Not Found.
oh-my-posh: 21.27.0 (latest version)
Latest versions for all apps are installed! For more information try 'scoop status'

and also oh-my-posh upgrade:

 ❯ oh-my-posh upgrade

📦 Upgrading Oh My Posh from 21.27.0 to v24.13.0

🌎 Downloading latest version

❌ failed to download asset: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v24.13.0/posh-windows-amd64.exe

Where install file names changed? I don't see this file on the releases page. 

Thanks.

**Comment by JanDeDobbeleer** [comment]
@PilotBob something went wrong with the release due to a broken dependency, I'm fixing that ATM.

**Reply by JanDeDobbeleer** [reply]
should be fixed now

**Reply by PilotBob** [reply]
yep thanks.

**Reply by ziyuang** [reply]
`scoop` doesn't work and seems to be keeping another version:
```
~ scoop update oh-my-posh
WARN  error: The remote server returned an error: (404) Not Found.
oh-my-posh: 21.17.1 (latest version)
```
vs.
```
~ oh-my-posh upgrade

📦 Upgrading Oh My Posh from 24.6.3 to v24.18.1

🚀 Upgrade successful, restart your shell to take full advantage of the new functionality.
```
And `scoop update` reports the same error message after the upgrade.

**Reply by JanDeDobbeleer** [reply]
@ziyuang that scoop bucket isn't maintained by me.

**Comment by RygelXVI** [comment]
I've been seeing the same problem. Winget reports version 24.13.1 is available, but will consistently produce a `NoApplicableUpdate` error if I try to update the package via winget (e.g. `Update-WinGetPackage -Id JanDeDobbeleer.OhMyPosh`).

Trying to directly update via `oh-my-posh upgrade` seems to not produce any reaction from either the terminal nor the computer in general.  Not sure if this is failing silently or something else is happening.

Having the configuration specify auto upgrade (i.e. `"auto_upgrade": true`) also does not seem to work.

I currently have 24.12.0 installed (at least according to WinGet).

**Reply by JanDeDobbeleer** [reply]
@RygelXVI I can't solve winget's issue but if you run `oh-my-posh upgrade`, that should also upgrade it. Be aware this can make winget and your oh-my-posh version out of sync.

**Reply by RygelXVI** [reply]
thanks for replying so quickly, and pointing me in the right direction.  I checked again, and this is a problem of WinGet and oh-my-posh getting out of sync. I've got the latest oh-my-posh installed, but WinGet can't see this... I'm guessing this will resolve itself with the next version of oh-my-posh.

**Reply by PilotBob** [reply]
I've had issues with winget with other stuff. Like it can't upgrade pwsh. I always have to uninstall it manually then re-install with winget. 

This is why I generally prefer scoop. At least for more dev/cli tools.
